packages/internet-header/src/services/route.service.spec.ts

```diff
@@ -1,3 +1,4 @@
+import { expect } from '@jest/globals';
```

Comment:
Can you configure ESLint so that this `@jest/globals` import can be removed? 
https://jestjs.io/docs/getting-started#using-eslint